pkg/capacity/planner_test.go

```diff
@@ -0,0 +1,200 @@
+package capacity
+
+import (
+	"testing"
+
+	"github.com/threefoldtech/tfexplorer/models/generated/workloads"
+	"github.com/threefoldtech/tfexplorer/pkg/capacity/types"
+)
+
+func Test_usesExpiredResources(t *testing.T) {
```

Comment:
Let's add all type of workloads in the test case so we cover all the possibility.